README.md

```diff
@@ -232,6 +232,12 @@ The Lambda wrapper adds the following dimensions to all data points sent to Sign
 
 1. Run `node-lambda deploy -f deploy.env` to deploy to AWS, which will use any environment variables configured in `.env`. For example: 
 
+#### Test in IntelliJ IDEA
```

Comment:
let's call this "Unit Tests in IntelliJ IDEA" just so it's clear that we have 2 ways of testing